index.html

```diff
@@ -21,6 +21,10 @@
     <br>
     <a style="margin-right: 1em;" href="https://github.com/Bitcoin-ABC/bitcoin-abc">Source Code</a>
     <a style="margin-left: 1em;" href="https://download.bitcoinabc.org/">Binaries</a>
+    <br>
```

Comment:
Something is missing to separate the "official" binaries and the distro packages. May something like this?
`<span style="font-weight: bold">Packages:</span>`

Reply:
I like this suggestion, just need to make sure it looks good =)